client/state/reader/posts/normalization-rules.js

```diff
@@ -64,11 +64,14 @@ function discoverFullBleedImages( post, dom ) {
  * @return {object}            The classified post
  */
 function classifyPost( post ) {
-	var displayType = DISPLAY_TYPES.UNCLASSIFIED,
-		canonicalImage = post.canonical_image,
+	const canonicalImage = post.canonical_image;
+	let displayType = DISPLAY_TYPES.UNCLASSIFIED,
 		canonicalAspect;
 
-	if ( post.images && post.images.length === 1 && post.images[0].naturalWidth >= PHOTO_ONLY_MIN_WIDTH && post.word_count < 100 ) {
+	if ( post.images &&
```

Comment:
`client/lib/feed-post-store/...` is going to go away entirely in the near future. i'm not worried about duping the logic for now.